src/main/markdown/doc/latest/DevGuideCodingBasicsJsInterop.md

```diff
@@ -3,6 +3,7 @@ JsInterop
 
 JsInterop is one of the core features of GWT 2.8. As the name suggests, JsInterop is a way of interoperating Java with JavaScript. It offers a better way of communication between the two using annotations instead of having to write JavaScript in your classes (using JSNI). More details about the annotations can be found in GWT javadoc: http://www.gwtproject.org/javadoc/latest/jsinterop/annotations/package-summary.html
 
+NOTE: You need the `-generateJsInteropExports` GWT compiler argument
```

Comment:
This is not true. `-generateJsInteropExports` is only needed in the "exporting a Java type to JavaScript" case; so maybe add something to that section specifically instead.

And maybe also document `-includeJsInteropExport` and `-excludeJsInteropExport`?